src/pages/BaseSwagLab.page.js

```diff
@@ -1,14 +1,33 @@
 const { BasePage } = require('./Base.page');
 
 export class BaseSwagLabPage extends BasePage {
-    // header
-    get mainMenuBtn() { return this.page.locator('TBD'); }
-
-    get shopingCart() { return this.page.locator('.shopping_cart_link'); }
+    constructor(page) {
+        super(page);
+        this.shopingCartBadge = this.page.locator('.shopping_cart_badge');
+        this.shopingCartLink = this.page.locator('.shopping_cart_link');
+        this.items = this.page.locator('.cart_item');
+    }
 
-    get shopingCartBadge() { return this.page.locator('.shopping_cart_badge'); }
+    async openShoppingCart() {
+        return this.shopingCartLink.click();
+    }
 
     async getNumberOfItemsInCart() {
         return this.shopingCartBadge.textContent();
     }
+
+    async getItemsList() {
```

Comment:
```suggestion
    async getItemsListData() {
```
I'd rather rename it for better understanding of the result of this function